build.gradle

```diff
@@ -7,35 +7,35 @@ buildscript {
 
     ext {
         versions = [
-                'androidGradlePlugin': '4.1.0',             // https://developer.android.com/studio/releases/gradle-plugin.html
+                'androidGradlePlugin': '4.2.0-beta06',             // https://developer.android.com/studio/releases/gradle-plugin.html
```

Comment:
Any particular reason for the beta? Compose support? 

Also nit, 👾!

Reply:
Ack! I will definitely fix that space

Also, why beta? 'Cause that's what I'm using :)
Compose support isn't official until 4.3.